reconcile/gql_definitions/terraform_cloudflare_resources/terraform_cloudflare_resources.gql

```diff
@@ -2,6 +2,43 @@
 query TerraformCloudflareResources {
   namespaces: namespaces_v1 {
     name
+    clusterAdmin
+    cluster {
+      name
+      serverUrl
+      insecureSkipTLSVerify
+      jumpHost {
+        hostname
+        knownHosts
+        user
+        port
+        identity {
+          path
+          field
+          version
+          format
+        }
+      }
+      automationToken {
+        path
+        field
+        version
+        format
+      }
+      clusterAdminAutomationToken {
+        path
+        field
+        version
+        format
+      }
+      spec {
+        region
+      }
+      internal
+      disable {
+        integrations
```

Comment:
add `e2eTests` -> This will make the protocol happy.

Reply:
Fixed and as you mentioned the type error resolved. Thanks!